src/Microsoft.Azure.WebJobs.Host.Storage/IAzureStorageProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Note that consumers of webjobs will now have to provide a concrete implementation for this because we do not provide one in WebJobs.Host.Storage.

Reply:
I don't understand - calling AddAzureStorageCoreServices should be sufficient to register a default implementation for this.

Reply:
Need to provide default internal implementation